util/cli/cli.go

```diff
@@ -67,12 +67,21 @@ func AddKubectlFlagsToCmd(cmd *cobra.Command) clientcmd.ClientConfig {
 // AddKubectlFlagsToSet adds kubectl like flags to a provided flag set and returns the ClientConfig interface
 // for retrieving the values.
 func AddKubectlFlagsToSet(flags *pflag.FlagSet) clientcmd.ClientConfig {
+	return AddKubectlFlagsToSetInCore(flags, false, "")
+}
+
+// AddKubectlFlagsToSetInCore adds kubectl like flags to a provided flag set and returns the ClientConfig interface for retrieving the values and support Core mode to set custom kubeconfig file location.
+func AddKubectlFlagsToSetInCore(flags *pflag.FlagSet, coreMode bool, kubeConfigLocation string) clientcmd.ClientConfig {
```

Comment:
If AddKubectlFlagsToSet function does nothing but call AddKubectlFlagsToSetInCore function, then why not implement it inside caller function and save one function call?